### DISCUSSION TITLE
Flac Radio

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1160] [Feature Requests]
First of all I want to thank Rern for all of his work . Meanwhile my Systems are running nearly without any bugs . The biggest bug was me , because of using a sd-card extension-cable that was causing most of the bugs I had . I'm using an
USB-Stick for boot now and all is running fine . 
My question now : Is it possible to integrate Interpret and Songtitle of Flac-Radio-Stations , because they are looking like this "http://amp.cesnet.cz:8000/cro-jazz.flac" at the moment.

**Comment by Raitsa** [comment]
Flac song infos working quite nicely

https://i.imgur.com/xaVKqpr.png

https://i.imgur.com/wwYcNvw.png

**Comment by hjfisch** [comment]
Surely but not with Internet Radio Stations sending in Flac instead of MP3 (f.e. Naim Jazz , Cesky Jazz ,Hi On Line ...)

**Comment by Raitsa** [comment]
Are you saying that the above radio paradise flacs are not flacs?

**Comment by cmh714** [comment]
Yes, flac streaming works perfectly when the stations provide the correct information.....

The links from the https://amp.cesnet.cz/ page are interesting. Those links are actually completely full files of the actual music, and that is not how streaming is done IMHO. Not sure if there is a way to have that parsed such that you see the Artist, Title, etc.

Maybe one of the other users or @rern has ideas... GL!

**Comment by R101G** [comment]
If a station does not include metadata which matches what has been uploaded to a public database, rAudio will not find any info or cover art. It might be worth checking the character set in some cases. 

Radio Paradise has its own database.
